en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/security/api-authentication/api-key-authentication.md

```diff
@@ -130,8 +125,65 @@ Use one of the following options to invoke APIs using an API key based on the me
 
 ## Additional Information
 
-### API key restriction for IP address and HTTP referrer
-
 Choreo Connect API Key Authentication supports API key restriction for IP address and HTTP referrer. 
 
 {!includes/design/additional-api-key.md!}
+
+## Configurations related to API Key
+
+You may use the following to reconfigure API Authentication related configs.
+
+```
+[[enforcer.security.tokenService]]
+name = "APIM APIkey"
+validateSubscription = true
+certificateAlias = "apikey_certificate_alias"
+certificateFilePath = "/home/wso2/security/truststore/wso2carbon.pem"
+```
+
+<table>
+  <tr>
+    <th>Parameter</th>
+    <th>Description</th>
+  </tr>
+  <tr>
+    <td>name</td>
+    <td>Name of the token service and must not be changed</td>
+  </tr>
+  <tr>
+    <td>certificateAlias</td>
+    <td>Alias name to be used by the internal Enforcer truststore for the public certificate of the JWT issuer</td>
+  </tr>
+  <tr>
+    <td>certificateFilePath</td>
+    <td>Certificate file path within the Enforcer container</td>
+  </tr>
+  <tr>
+    <td>validateSubscription</td>
+    <td>
+    Whether to validate subscriptions. **If disabled**, a single API key may be used against any API in the same Choreo Connect instance since only the JWT signature would be validated procedure. When set to true, sunscription validation is done as given below depending on the Choreo Connect mode. </br></br>
```

Comment:
```suggestion
    Whether to validate subscriptions. **If disabled**, a single API key may be used against any API in the same Choreo Connect instance since only the JWT signature would be validated procedure. When set to true, subscription validation is done as given below depending on the Choreo Connect mode. </br></br>
```